config/kubernetes/staging/prometheus.yml

```diff
@@ -6,15 +6,15 @@
 # in both, staging and production environments.
 #
 # To see the current alerts in this namespace:
-#   kubectl describe prometheusrule -n laa-review-criminal-legal-aid-production
+#   kubectl describe prometheusrule -n laa-review-criminal-legal-aid-staging
 #
 # Alerts will be sent to the slack channel: #laa-crime-apply-alerts
 #
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  name: prometheus-production
-  namespace: laa-review-criminal-legal-aid-production
+  name: prometheus-staging
```

Comment:
As we only have 1 prometheus per namespace, the name can be made generic `prometheus-rules`
We have this name in the other services.